src/api.ts

```diff
@@ -35,6 +37,32 @@ export const bas = {
         return ActionsController.getAction(actionId);
     },
 
+    async getParameter(parameterName: string) : Promise<string | null> {
+        const onBasEnv = "env.WS_BASE_URL";
+        const isInBAS = !_.isEmpty(_.get(process, onBasEnv));
+        if (!isInBAS) {
+            getLogger().trace("Running in VS Code, so returning null.");
+            return null;
+        }
+        getLogger().trace("Running on BAS.");
+
+        let sap;
+        try {
+            sap = require('@sap/plugin');
```

Comment:
try to use `https://www.npmjs.com/package/optional-require`

Reply:
When I use optional-require, I don't need to test isInBAS, right? Are you ok I'll remove it?

Reply:
not sure, lets sync offline